dev-build/conf/routes

```diff
@@ -411,6 +411,7 @@ GET            /$path<[\w\d-]*(/[\w\d-]*)?/gallery/.*>
 GET            /$path<[\w\d-]*(/[\w\d-]*)?/(cartoon|graphic|picture)/.*>.json                                                    controllers.ImageContentController.renderJson(path)
 GET            /$path<[\w\d-]*(/[\w\d-]*)?/(cartoon|graphic|picture)/.*>                                                         controllers.ImageContentController.render(path)
 
```

Comment:
This file: https://github.com/guardian/frontend/blob/master/standalone/conf/standalone.routes#L139 Do you need to add your new route there? IIRC standalone is used for preview or something - @rich-nguyen ?